products/ubuntu2404/product.yml

```diff
@@ -0,0 +1,54 @@
+product: ubuntu2404
+full_name: "Ubuntu 24.04"
+type: platform
+
+families:
+  - debian-like
+  - ubuntu
+
+major_version_ordinal: 2404
+
+benchmark_id: UBUNTU_24-04
+benchmark_root: "../../linux_os/guide"
+components_root: "../../components"
+
+profiles_root: "./profiles"
+
+pkg_manager: "apt_get"
+pkg_manager_config_file: "/etc/apt/apt.conf"
+
+init_system: "systemd"
+oval_feed_url: "https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.jammy.cve.oval.xml"
```

Comment:
@dodys Is this the correct URL? It doesn't match what is in the product stability.